x-pack/test/plugin_functional/test_suites/global_search/global_search_providers.ts

```diff
@@ -14,68 +14,84 @@ export default function ({ getPageObjects, getService }: FtrProviderContext) {
   const browser = getService('browser');
   const esArchiver = getService('esArchiver');
 
-  const findResultsWithAPI = async (t: string): Promise<GlobalSearchResult[]> => {
+  const findResultsWithApi = async (t: string): Promise<GlobalSearchResult[]> => {
```

Comment:
These tests aren't running now but will once this FTR suite is fixed from https://github.com/elastic/kibana/pull/83422